customer/models.py

```diff
@@ -1,16 +1,20 @@
 from django.db import models
 
-from additional.models import BaseData
+from abstract.abstract_models import BaseData
 from auto_show.models import DiscountAutoShow
 
 
 class Customer(BaseData):
-    """Model for Customer"""
-
     last_name = models.CharField(max_length=200, verbose_name="Фамилия")
     first_name = models.CharField(max_length=200, verbose_name="Имя")
     year_of_birth = models.PositiveIntegerField(blank=True, null=True, verbose_name='Год рождения')
     balance = models.IntegerField(default=0, verbose_name='Баланс, USD')
```

Comment:
Why did you use IntegerField for money? What about cents?
It's better to use DecimalField here

Reply:
+